### PR TITLE
Feat/home page

### PR DESCRIPTION
**Pull request by bhavyaj12**
### Added Home Page, and Auth dummy pages, Router

- I have added the home page using my UI Lib
- I have added react router 
- Login page, Signup Page
- Navigation and Footer Components
- **Pages are responsive**

### Files to check

- [ ] src/pages/HomePage/HomePage.jsx
- [ ] src/components/Footer/Footer.jsx
- [ ] src/components/NavTop/NavTop.jsx
- [ ] src/pages/Authentication/LoginPage.jsx
- [ ] src/pages/Authentication/SignupPage.jsx

**NOTE: Ignore other files, they're index, images, styles**

### What not to check

- Authentication Functionality

image
image

### CODE REVIEW

**Review by rishichourasia — APPROVED** [review]
Not Much to Review since its just components and not logic. looks good as far to me. 
really like your design sense captain.homepage is so nice . Maybe change your footer bg colour to something lighter shade for UI prespective . looking forward to review more functionalaties.

File: src/pages/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            Scribble down all your unique and amazing ideas, give yourself a
+            productivity boost by managing your task list and more, using
+            Scriptus.
+          </div>
+          <div className="button-landing">
+            <a href="">
+              <button className="btn-notes-home btn-primary">
+                Start Taking Notes
+              </button>
+            </a>
```
Comment (harshitpaliwal95) [inline src/pages/HomePage/HomePage.jsx:29]: remove href its reloading page on click

**Review by harshitpaliwal95 — COMMENTED** [review]
- everything looks good you can merge 
- added one suggestion

File: src/pages/HomePage/HomePage.jsx (hunk 1 above)
Comment (bhavyaj12) [reply src/pages/HomePage/HomePage.jsx:29]: Yes will remove when I add auth functionality

**Pull request merged**